src/praisonai/praisonai/cli/commands/pairing.py

```
@@ -0,0 +1,181 @@
+            label=label
+        )
+        
+        if success:
+            click.echo(f"✅ Successfully paired {platform} channel {channel_id}")
+            if label:
+                click.echo(f"   Label: {label}")
+        else:
+            click.echo(f"❌ Failed to pair - invalid or expired code: {code}", err=True)
+            raise click.ClickException(f"Invalid code: {code}")
+            
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)